### ISSUE TITLE
LOW PRIORITY    Trying to show my hunts from newest to oldest

### ISSUE THREAD

**Issue by Loganwebb1994** [nss-day-cohort-46/How-to-Ask-for-Help#227]
**_Ticket MUST be completed before getting help from an instructor_**
#### Have you searched the other issue tickets? If not, do that first. 
- [x] Yes
- If No, stop and search.

####  REQUIRED Add Tags to Issue Ticket and Assign to Project Board
 

- [x] Added to `c46 Help-Thread` project board
- [] Added the `Blocker` label if this is stoping you from moving forward

#### REQUIRED Describe your issue:
>> I know that I can add ```?_sort=timestamp``` to have the objects my fetch returns be in order. I know there is a way to sort it by ascending or descending order, but I don't remember the syntax and am having trouble finding anything online. 

#### REQUIRED What have you googled? **_(You must have at least 3 links related to your issue)_**
- [x] 1. **https://www.algolia.com/doc/guides/managing-results/refine-results/filtering/how-to/filter-by-date/**
- [x] 2.  **https://stackoverflow.com/questions/7555025/fastest-way-to-sort-an-array-by-timestamp**
- [ ] 3. 

#### REQUIRED: What have you tried? Tell us everything you have tried. 
>>Ive tried reviewing old code and I tried to find away to pull my postman history because Im pretty scott and I played around with it at some point.

#### REQUIRED: Copy/paste link TO THE FILE on your branch that you are having issues with

**https://github.com/Loganwebb1994/oh-deer/blob/main/src/components/stands/StandProvider.js**

#### Code Snippet

```
const getHunts = () => {
        return fetch("http://localhost:8088/userStands?_expand=stand?_sort=timestamp")
        .then(res => res.json())
        .then(res => setHunts(res))
      }

```

**Comment by Scott47** [comment]
@Loganwebb1994 I am assuming this is done. I remember discussing using `_sort=timestamp` on fetch.

**Comment by Scott47** [comment]
```
const sortedHunts = hunts.sort(
    (currentHunt, nextHunt) => (nextHunt.timestamp) - (currentHunt.timestamp)
  )
  ```
